data/slime_chunk_finder/predicates/has_item.json

```
@@ -0,0 +1,29 @@
+{
+  "condition": "minecraft:alternative",
+  "terms": [
+    {
+      "condition": "minecraft:entity_properties",
+      "entity": "this",
+      "predicate": {
+        "equipment": {
+          "mainhand": {
+            "item": "minecraft:slime_ball",
+            "nbt": "{HideFlags:1,Enchantments:[{id:\"minecraft:unbreaking\",lvl:1s}],display:{Name:'{\"text\":\"Slime Chunk Finder\",\"italic\":false}'}}"
```

Does this have to match the item exactly, or could we simplify it using a single custom tag?
```suggestion
            "nbt": "{slimeChunkFinder:1b}"
```
This seems like it would be significantly cleaner, and would even allow the player to rename the item. 
Lore could be added as well to retain what the item is despite a rename.

---

I'm not a huge fan of adding custom nbt keys, however the name restriction can be removed (given that it's impossible to obtain an enchanted slimeball in Survival).